src/helpers/format-time-left.js

```diff
@@ -0,0 +1,7 @@
+const formatTimeLeft = (timeLeft) => {
```

Comment:
that's okay but do you remember the case of the `Backspace` component that eventually accepted `disabled` props and initially internally dealt with the logic related to `slots`? in that case we ended up with separation of concerns as `Backspace` component gets `disabled` prop only

here it's a bit similar but more subtle one and I don't say you **have** to change the code but I would like you to at least consider the issue: this function is formatting seconds into a pretty string. Now the "seconds" happen to be "time left" but in general it can be anything so if you want to make this function even more reusable it would make sense to name it `formatDuration` (or similarly).

also, this function is a perfect opportunity write unit tests!

Reply:
> that's okay but do you remember the case of the `Backspace` component 

I remember. but are you refering to naming only or sth more?

> it would make sense to name it `formatDuration` (or similarly).

I would go for `formatTime` then. `duration` doesn't apply e.g. to couting time down to a specific date in the future (and this is sth I know too well from elsewhere 😁).

> also, this function is a perfect opportunity to write unit tests!

🎉 let's do this! **how?** 😁

Reply:
here in `formatTimeLeft` case I meant the issue of naming only

have a look at https://create-react-app.dev/docs/running-tests
there are sections that are irrelevant to you but perhaps it's a got intro

what's your experience with testing? I don't know what you know

in this project we dealt with a test file right in the beginning, fixing `App` import issue ;) so there is a test file in the project and it's configured by create react app, all you have to do is to write tests

Reply:
ok, I feel convinced to `duration` :)

tests - I've got some experience with Jest, I think I'll manage :) how about a separate PR?
you mention `App.test.js` - you want me to write it there? 🤔 (I thought test files should be close to what they test).

Reply:
Separate PR - sure, next PR :)
If you test x.js then your tests go to x.test.js (this is just a naming convention, in general it can be named differently, other popular suffix is "spec", it depends on how project is configured)

Reply:
> I thought test files should be close to what they test

:+1: :clap: :fireworks:

I just mentioned App test file to tell the process of running test files is configured out of the box by create react app setup.